packages/pds/tests/migrations/repo-sync-data.test.ts

```diff
@@ -1,82 +1,153 @@
-import AtpApi from '@atproto/api'
 import { Database } from '../../src'
-import { SeedClient } from '../seeds/client'
-import { CloseFn, runTestServer } from '../_util'
-import basicSeed from '../seeds/basic'
-import { RepoCommitHistory } from '../../src/db/tables/repo-commit-history'
-import { RepoCommitBlock } from '../../src/db/tables/repo-commit-block'
+import { MemoryBlockstore, Repo, WriteOpAction } from '@atproto/repo'
+import { EcdsaKeypair, Keypair, randomStr } from '@atproto/crypto'
+import { TID } from '@atproto/common'
+import { Kysely } from 'kysely'
+import { CID } from 'multiformats/cid'
 
 describe('repo sync data migration', () => {
-  let close: CloseFn
   let db: Database
+  let rawDb: Kysely<any>
+  let memoryStore: MemoryBlockstore
+  let keypair: Keypair
+  let repo: Repo
+
+  const aliceDid = 'did:example:alice'
+  const bobDid = 'did:example:bob'
 
   beforeAll(async () => {
-    const server = await runTestServer({
-      dbPostgresSchema: 'migration_repo_sync',
-    })
-    close = server.close
-    db = server.ctx.db
-    const client = AtpApi.service(server.url)
-    const sc = new SeedClient(client)
-    await basicSeed(sc, server.ctx.messageQueue)
+    if (process.env.DB_POSTGRES_URL) {
+      db = Database.postgres({
+        url: process.env.DB_POSTGRES_URL,
+        schema: 'migration_duplicate_records',
```

Comment:
```suggestion
        schema: 'migration_repo_sync_data',
```

Reply:
my god dude. one of these days ill get it right 🤦